### PR TITLE
added some project ideas

### PR DESCRIPTION
**Pull request by aman34503**

### CODE REVIEW

**Comment by aman34503** [comment]
When will this Request get merged

**Comment by vikasbhalla05** [comment]
I'm thinking the same thing
@aman34503

**Comment by praveenscience** [comment]
@vikasbhalla05 Yours has been merged.
@aman34503 I have a full time job, and there are 20+ PRs now to review. I am doing it one by one. Doing it once per week. Hope that clarifies your question.

**Review by praveenscience — CHANGES_REQUESTED** [review]
Few changes. Sorry.

**Comment by aman34503** [comment]
> @vikasbhalla05 Yours has been merged.
> @aman34503 I have a full time job, and there are 20+ PRs now to review. I am doing it one by one. Doing it once per week. Hope that clarifies your question.

Sorry, i didn't mean this, i am only asking.

File: README.md [hunk 1]
```diff
@@ -271,6 +276,7 @@ We'll organise into different streams here. One kind request for motivation: Ple
 * QR Code Generator
 * Cone and Obstacle Detection
   > _using yolov5/open cv_
+  * URL shortener
```
[the remark this answers is not in the document]
Comment (aman34503) [reply README.md]: done.

**Comment by praveenscience** [comment]
@aman34503 Yes I know. Don't worry, didn't take it in the wrong sense man. 😁

**Review by praveenscience — APPROVED** [review]
👍

**Comment by praveenscience** [comment]
@aman34503 Thanks for your contribution. Optional Request: [Subscribe to my **YouTube Channel**](https://www.youtube.com/praveenscience?sub_confirmation=1), if you haven't already subscribed and share it with your friends too.

**Pull request merged**